### ISSUE TITLE
I am getting the same prediction value for all of my test data

### ISSUE THREAD

**Issue by shreyassks** [tensorflow/tensorflow#23021]
I tried with many combinations of hyper parameters, optimizers and cost functions but I'm getting R2 = 0.33 at max. Please help me finding a solution.
[Python code.txt](https://github.com/tensorflow/tensorflow/files/2482551/Python.code.txt)
[Gholamnejad_dataset1.xlsx](https://github.com/tensorflow/tensorflow/files/2482564/Gholamnejad_dataset1.xlsx)

**Comment by tensorflowbutler** [comment]
Thank you for your post. We noticed you have not filled out the following field in the issue template. Could you update them if they are relevant in your case, or leave them as N/A? Thanks.
Have I written custom code
OS Platform and Distribution
TensorFlow installed from
TensorFlow version
Bazel version
CUDA/cuDNN version
GPU model and memory
Exact command to reproduce
Mobile device

**Comment by harshini-gadige** [comment]
@shreyassks Hi, I'd request you to perform scaling/data preprocessing for the input data. Also try reducing the learning rate and reverify. Thanks !

**Comment by alextp** [comment]
This should be a stackoverflow question, not a github issue. Please ask there.
